client/src/api/requests/competitions.ts

```diff
@@ -29,3 +30,23 @@ export const getCompetitionParticipantsApi = async (id: string) => {
 
 	return result
 }
+
+export const getCompetitionJudgesApi = async (id: string) => {
```

Comment:
U need to get all judges from the data base and show them. If competition already has two judges connected, then u can't add judges again.